pkg/client/client.go

```diff
@@ -230,6 +235,14 @@ var (
 	_ Client         = (*DynatraceClient)(nil)
 )
 
+// WithClientRequestLimiter specifies that a specifies the limiter to be used for
+// limiting parallel client requests
+func WithClientRequestLimiter(limiter *concurrency.Limiter) func(client *DynatraceClient) {
```

Comment:
with the face that `concurrency.Limiter` is from internal package, is it going to be possible to use this as a library?

Reply:
inside our code-base, yes. from another module, no